Formula/lxi-tools.rb

```diff
@@ -0,0 +1,31 @@
+class LxiTools < Formula
+  desc "Open source tools for managing network attached LXI compatible instruments"
+  homepage "https://github.com/lxi-tools/lxi-tools"
+  url "https://github.com/lxi-tools/lxi-tools/archive/refs/tags/v2.5.tar.gz"
+  sha256 "e8ccefeb907da1dacecef2c5b8bd0c1a841003a4d1309143426f860ff8c14bc0"
+  license "BSD-3-Clause"
+
+  head "https://github.com/lxi-tools/lxi-tools.git", branch: "master"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+
+  depends_on "desktop-file-utils"
+  depends_on "glib"
+  depends_on "gtk4"
+  depends_on "gtksourceview5"
+  depends_on "json-glib"
+  depends_on "libadwaita"
+  depends_on "liblxi"
+  depends_on "lua"
+
+  def install
+    system "meson", "setup", "build", *std_meson_args
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build"
+  end
+
+  test do
+    system "#{bin}/lxi", "-v"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.